### PR TITLE
Misc gPTP changes part 1

### PR DESCRIPTION
**Pull request by jukkar**

### CODE REVIEW

**Review by pfalcon — APPROVED** [review]
Eyeballed, approving to not stall gPTP process.

File: include/net/gptp.h [hunk 1]
```diff
@@ -165,17 +165,10 @@ struct gptp_hdr {
...
 
-/*
- * TODO: k_uptime_get need to be replaced by the MAC ptp_clock.
- * The ptp_clock access infrastructure is not ready yet
- * so use it for the time being.
- * k_uptime time precision is in ms.
- */
-#define GPTP_GET_CURRENT_TIME_NANOSECOND() (k_uptime_get() * 1000000)
-#define GPTP_GET_CURRENT_TIME_USCALED_NS(uscaled_ns_ptr)		\
+#define GPTP_GET_CURRENT_TIME_USCALED_NS(port, uscaled_ns_ptr)		\
```
Comment (pfalcon) [inline include/net/gptp.h]: Though wait, what is "uscaled" in "USCALED_NS"?

Comment (jukkar) [reply include/net/gptp.h]: From IEEE 802.1AS spec: "The UScaledNs type represents unsigned values of time and time interval in units of 2 – 16 ns."

Comment (pfalcon) [reply include/net/gptp.h]: Thanks

**Pull request merged**